src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,25 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private Integer power;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(Integer power, String name, String color) {
+        this.power = power;
+        setName(name);
+        setColor(color);
```

Comment:
don't call methods inside of the constructor, let's call constructor of the parent, the same for other machines